packages/react-composites/package.json

```diff
@@ -54,7 +44,7 @@
     "lint:quiet": "npm run lint -- --quiet"
   },
   "dependencies": {
-    "@azure/communication-calling": "1.3.2",
+    "@azure/communication-calling": "1.3.2-beta.1 || 1.3.2",
```

Comment:
fyi in case this is completed first: #1294

Reply:
Actually, @PorterNan how will rush variants work with peerDependencies and devDepenedencies?

Thanks for linking to my PR James.

Reply:
My understanding is that rush variants work at the level of pnpm. So it doesn't matter why we try to install a version, it just switches out the version actually installed. But it'd be worth checking.

Reply:
Will merge the main branch and take a test then

Reply:
Yea, it works when peerDependencies are stable version, but devDependencies with "1.3.2-beta.1 || 1.3.2"